tests/Behat/Tests/Config/ConfigTest.php

```diff
@@ -20,7 +20,7 @@ public function testItReturnsSettings(): void
             'default' => [
                 'gherkin' => [
                     'filters' => [
-                        'tags' => '~@php8'
+                        'tags' => '~@test'
```

Comment:
This change is not strictly necessary but I thought it would be better to remove any mention to PHP 8